pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -90,12 +90,12 @@
       "SELECT COUNT(*) FROM mytable WHERE DivActualElapsedTime = 305";
 
   // For range index triggering test
-  private static final List<String> UPDATED_RANGE_INDEX_COLUMNS = Collections.singletonList("DivActualElapsedTime");
+  private static final List<String> UPDATED_RANGE_INDEX_COLUMNS = Arrays.asList("Origin", "DivActualElapsedTime");
```

Comment:
We might want to test add/remove at the same time?

Reply:
thought about that, but it'd be tricky to assert on table size if add+delete happens together.

Reply:
Understood. IMO we can skip the size check and only check the size after the indices are changed back